src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -3321,6 +3050,39 @@ internal static void SetConsoleCursorInfo(ConsoleHandle consoleHandle, CONSOLE_C
 
 #endregion helper
 
+#region 
+
+        private static int LengthInBufferCells(char c)
+        {
+            if (c < 256)
```

Comment:
This condition only applies to PSReadline.

Reply:
will remove